components/Navigation/index.js

```diff
@@ -0,0 +1,70 @@
+import Link from "next/link";
+import styled from "styled-components";
+import ProductSvg from "@/public/images/products_icon.svg";
+import StoreSvg from "@/public/images/stores_icon.svg";
+
+import { useRouter } from "next/router";
+
+const StyledNavBar = styled.ul`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin: 0;
+`;
+
+const StyledNavLink = styled(Link)`
+  text-decoration: none;
+  display: inline-block;
+  padding: 10px;
+  width: 100%;
+  border-top: ${({ $currentPage }) =>
+    $currentPage
+      ? "solid 3px var(--accentColor)"
+      : "solid 3px var(--secondaryBackgroundColor)"};
+  &:hover {
+    border-color: ${({ $currentPage }) => !$currentPage && "gray"};
+  }
```

Comment:
I like the way you style your navigation ❤️

Reply:
Thanks! 🥰